### PR TITLE
✨ — Add bigImageUrl option to push command

### PR DESCRIPTION
**Pull request by ldidry**
Warning: I did not compile to try this as I don’t have a GO dev env, but I think there’s should be no problem.

### CODE REVIEW

**Comment by ldidry** [comment]
I just found that the `Dockerfile` section of README.md contains instructions to build the application without installing a lot of things to your system, so I was able to build the app.

`--bigImageUrl` works as expected 🙂

File: command/push.go [hunk 1]
```diff
@@ -95,6 +97,12 @@ func doPush(ctx *cli.Context) {
 		}
 	}
 
+	if bigImageUrl != "" {
+		msg.Extras["client::notification"] = map[string]interface{}{
```
Comment (jmattheis) [inline command/push.go:101]: With this, it won't be possible to supply both --clickUrl and --bigImageUrl together, as bigImageUrl currently overrides clickUrl.

Comment (ldidry) [reply command/push.go]: You’re right. Fixed in the new version I just forced push (tested and verified)

**Review by jmattheis — APPROVED** [review]
Thanks

**Comment by ldidry** [comment]
Thanks to you! I don’t know a thing about golang :sweat_smile:

**Pull request merged**